### DISCUSSION TITLE
Hide the edit button

### DISCUSSION THREAD

**Discussion by kathy-li-ho** [tinacms/tinacms#2233] [Help]
Hello! I can't seem to hide the edit button?
- I know I need to set `NEXT_PUBLIC_HIDE_EDIT_BUTTON=1` but that doesn't work. 
- I also straight up set `showEditButton={false}` in the `TinaEditProvider`, but it's still there! 

I restart the server and everything. Any suggestions?

**Comment by scottgallant** [comment]
Hi @kathy-li-ho  is it appearing both locally and in your Vercel deployment?

**Reply by kathy-li-ho** [reply]
oh good point. It works on the Vercel deployment, but not locally.

**Comment by logan-anderson** [comment]
Very strange, I just updated to the latest version and it seems to be working for me. What version of `tinacms` are you using?

Here is a video if it working for me: https://www.loom.com/share/79f6c1ffacc742b79b16031a65b1f6bd

**Reply by kathy-li-ho** [reply]
`"tinacms": "^0.56.0",`, and I was doing the same thing you did in your recording.

**Comment by kathy-li-ho** [comment]
Okay, so apparently code only works when I publically make a spectacle, so the edit button is hiding as it should now. My problem is now navigating to `/admin` or `/exit-admin` require a page refresh to toggle the button.
